js/graph_config.js

```diff
@@ -564,6 +564,7 @@ GraphConfig.load = function(config) {
         
         if (!flightLog.isFieldDisabled().MOTORS) {
             EXAMPLE_GRAPHS.push({label: "Motors",fields: ["motor[all]", "servo[5]"]});
+            EXAMPLE_GRAPHS.push({label: "Motors (Legacy)",fields: ["motorLegacy[all]", "servo[5]"]});
```

Comment:
I think remember that in the other PR discussion the final decision was to use the new Motor graph as default. In this case we are changing it to the Legacy one.

Reply:
This isn't setting any defaults - this is just adding a shortcut to allow users to add a legacy Motors graph if they wish to do so without requiring extra clicks:

![image](https://user-images.githubusercontent.com/4742747/100624973-e249fa80-3388-11eb-970a-686e43623d4c.png)

We are doing this for pretty much all other groups of graphs, so no reason to not do this for this group.

Reply:
True. I didn't notice it was an addition and not a change...